From 8b2bd9f75deef895e3c9f076419b22a7b1fa63aa Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Thu, 14 Dec 2023 15:54:39 +0100
Subject: [PATCH] Fix finding packages witch hatchling >= 1.19.0

Explicitly specify the package name to install, in order to fix
compatibility with `hatchling >= 1.19.0`.  Starting with this version
(though Frozen-Flask is affected since 1.20.0, due to a bug
in hatchling), it is necessary to explicitly select packages if they
do not

As of hatchling >= 1.19.0, it is necessary to explicitly specify
`packages` to use if they do not fit the default heuristics (i.e. are
disjoint from the project name).

Since the package is now specified explicitly, the `exclude` rules are
no longer necessary.
---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 2fdfefc..fcf6c1d 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -52,7 +52,7 @@ path = 'flask_frozen/__init__.py'
 exclude = ['.*']
 
 [tool.hatch.build.targets.wheel]
-exclude = ['docs', 'tests']
+packages = ['flask_frozen']
 
 [tool.hatch.envs.doc]
 features = ['doc']
